runtime/src/main/java/org/corfudb/runtime/collections/streaming/StreamStatus.java

```diff
@@ -0,0 +1,8 @@
+package org.corfudb.runtime.collections.streaming;
+
+public enum StreamStatus {
+    ERROR,
+    SCHEDULING,
+    RUNNABLE,
```

Comment:
Is this meant to be running? -> status?

Reply:
Nope. Runnable as in read to be executed, that is, scheduled and synced.